api/mmlogic.proto

```diff
@@ -54,24 +54,23 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
   // https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/proto/examplepb/a_bit_of_everything.proto
 };
 
-message RetrievePoolRequest {
+message GetPoolTicketsRequest {
   // The Pool representing the set of Filters to be queried.
   Pool pool = 1;
 }
 
-message RetrievePoolResponse {
+message GetPoolTicketsResponse {
   // The Tickets that meet the Filter criteria requested by the Pool.
   repeated Ticket ticket = 1;
 }
 
 // The MMLogic API provides utility functions for common MMF functionality such
 // as retreiving Tickets from state storage.
 service MmLogic {
-  // RetrievePool gets the list of Tickets that match every Filter in the
+  // GetPoolTickets gets the list of Tickets that match every Filter in the
   // specified Pool.
-  // TODO: Consider renaming to "GetPool" to be consistent with HTTP REST CRUD
-  // conventions. Right now there's a GET and a POST for this verb.
-  rpc RetrievePool(RetrievePoolRequest) returns (stream RetrievePoolResponse) {
+  // Right now there's a GET and a POST for this verb.
```

Comment:
Can we address this? This looks like a GET not a POST.

Reply:
The request contains a pool with repeated filters and it is not trivial and intuitive to include everything via a get url, so I leave the post binding for convenience purpose.